docs/source/autograd.rst

```diff
@@ -241,3 +241,15 @@ Anomaly detection
 .. autoclass:: detect_anomaly
 
 .. autoclass:: set_detect_anomaly
+
+
+Saved tensors default hooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some operations need intermediary results to be saved during the forward pass
+in order to execute the backward pass.
+You can define how these saved tensors should be packed / unpacked using hooks.
```

Comment:
Some more motivation here would be nice. E.g. "you might want to reduce memory if your model is just slightly bigger than the GPU memory you're using", "you might want to swap out to disk", etc.